touchforms/backend/xformplayer.py

```diff
@@ -445,7 +445,7 @@ def _parse_repeat_juncture(self, event):
         ro = r.getRepeatOptions()
 
         event.update(get_caption(r))
-        event['main-header'] = ro.header
```

Comment:
sorry yea, this affects the response of the question JSON. before one of the attributes was `main-header` now it's `header`. if you were accessing it anywhere and displaying it then it would break